source/content/guides/composer-convert.md

```diff
@@ -35,72 +30,82 @@ export site=yoursitename
 
 </Alert>
 
-## Checkout a New Branch
-You're about to make some massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back can be complicated:
+## Steps we'll take
+
+1. Convert to Composer
+1. Upgrade to Drupal 8.9
+1. Upgrade to Drupal 9
+
+## Add the Pantheon Integrated Composer Upstream in a New Local Branch
+
+You're about to make massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
 
 1. In your local terminal, change directories to your site project. For example, if you keep your projects in a folder called `projects` in the home directory:
 
   ```bash{promptUser:user}
   cd ~/projects/$site/
   ```
 
-2. Create the new branch:
+1. Add the Pantheon Drupal Upstream as a new remote, fetch the `ic` branch, and checkout to a new local branch based on it called `composerify`:
 
   ```bash{promptUser:user}
-  git checkout -b composify
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
   ```
 
-  You can replace `composify` with a branch name of your choosing, but all following steps assume this name.
-
-## Setup a Multidev (optional)
-If your Pantheon account has access to [multidev](/multidev), create a multidev to push your new code to:
-
-```bash{promptUser:user}
-git push origin composify && terminus env:create $site.dev composify
-```
+  You can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
 
-This will setup the multidev environment to receive and demo our changed code.
+1. Copy your configuration from the default branch:
 
-## Create a New Composer Project
+  ```bash{promptUser:user}
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git commit -m "Pull in configuration from default branch"
+  ```
 
-1. In your local terminal, from the repository root of your Pantheon site, move a directory up:
+1. Compare your `pantheon.yml` with the new `pantheon.upstream.yml` and check for conflicts:
```

Comment:
I didn't understand at first what this step was about, this context about what the goal is could help people to successfully interpret relevant things in the diff they get
```suggestion
1. Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
```